frames/src/main/java/com/checkout/frames/screen/paymentform/PaymentFormConfig.kt

```diff
@@ -12,5 +12,6 @@ public data class PaymentFormConfig @JvmOverloads constructor(
     public val environment: Environment,
     public var style: PaymentFormStyle,
     public val paymentFlowHandler: PaymentFlowHandler,
-    public var supportedCardSchemeList: List<CardScheme> = emptyList()
+    public var supportedCardSchemeList: List<CardScheme> = emptyList(),
+    public val cardHolderName: String? = null
```

Comment:
If we plan to have the billing address later, shall we wrap them in a data class and states it is for prefilling the form? For instance, 

```
public data class PaymentFormConfig @JvmOverloads constructor(
    public val publicKey: String,
    public val context: Context,
    public val environment: Environment,
    public var style: PaymentFormStyle,
    public val paymentFlowHandler: PaymentFlowHandler,
    public var supportedCardSchemeList: List<CardScheme> = emptyList(),
    public val prefixData: PrefillData? = null
)

// need a better naming
data class PrefillData(
  cardHolderName: String? = null,
  billingAddress: BillingAddress? = null
)
```

I know this would break the alignment with iOS API, but IMO, this way is more self-explain without reading any documentation or comments, compared to merely having a `cardholderName` or `billingAddress` in the config. What do you think? Let's also involve @patrick-hoban-cko for this.

Reply:
It is a valid suggestion and definitely, I would like to do it. However, we have been trying more focussing on the same naming convention for both platforms unless it is more platform-specific or architectural things. Happy to hear @patrick-hoban-cko thoughts here

Reply:
Let's have a quick chat on this guys, will ping you both

Reply:
As per our chat, let's go with:

```
// need a better naming
data class PrefillData(
  cardHolderName: String? = null,
  billingAddress: BillingAddress? = null
)
```